bash-scripts/create_tx.sh

```diff
@@ -20,26 +23,33 @@ then
         address="999999999999999999999999999999999999999999999999999999999999999999999999999999999"
 fi
 
-read -p "Please enter a trunk transaction hash: " trunk
-
-while [[ ! $trunk =~ ^[A-Z9]*{81}$ ]]; do
-        echo "Hash invalid. Transaction hashes much contain 81 trytes."
-	read -p "Please enter a trunk transaction hash: " trunk
-done
+#read -p "Please enter a trunk transaction hash: " trunk
+#
+#while [[ ! $trunk =~ ^[A-Z9]*{81}$ ]]; do
+#        echo "Hash invalid. Transaction hashes much contain 81 trytes."
+#	read -p "Please enter a trunk transaction hash: " trunk
+#done
+#
+#
+#read -p "Please enter a branch transaction hash: " branch
+#
+#while [[ ! $branch =~ ^[A-Z9]*{81}$ ]]; do
+#        echo "Hash invalid. Transaction hashes much contain 81 trytes."
+#        read -p "Please enter a branch transaction hash: " branch
+#done
 
 
-read -p "Please enter a branch transaction hash: " branch
+tips="$( node ../node-scripts/get-branch-and-trunk.js )"
```

Comment:
Is this way better than the way we do it in `do_pow.sh`?

We should use the same code in both

Reply:
Updated 😄